test/libsolidity/smtCheckerTests/functions/getters/inaccessible_dynamic_type_1.sol

```diff
@@ -0,0 +1,16 @@
+pragma experimental SMTChecker;
+contract C {
+	struct S {
+		string a;
+		uint256 y;
+	}
+	S public s;
+	function g() public returns (uint256) {
+		this.s();
+	}
+}
+// ====
+// EVMVersion: <=spuriousDragon
```

Comment:
Is the CI checking older versions of EVM? Does this make sense?

Reply:
Yea this is the right away. We need to check if CI is running old evm versions + smtchecker. I suspect it runs the smtchecker only with the latest evm version.